tools/msvs/msi/nodemsi.wixproj

```diff
@@ -54,9 +54,12 @@
     </WixExtension>
   </ItemGroup>
   <ItemGroup>
-    <!-- <EmbeddedResource Include="i18n\de-de.wxl" /> -->
-    <EmbeddedResource Include="i18n\en-us.wxl" />
+    <!--
```

Comment:
@joaocgreis the issue there is that WiX guys are going to make a new release only for the next Visual Studio release so it could take time

https://github.com/wixtoolset/wix3/pull/366#issuecomment-231612064